project.clj

```diff
@@ -4,12 +4,15 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :min-lein-version "2.0.0"
+  :exclusions [cljsjs/react]
   :dependencies [[org.clojure/clojure "1.9.0-alpha7"]
                  [org.clojure/clojurescript "1.9.76"]
                  [org.clojure/core.async "0.2.382"]
 ;;                  [org.clojure/tools.logging "0.3.1"] ;; Should remove this for timbre
                  [org.clojure/core.match "0.3.0-alpha4"]
                  [org.clojure/core.typed "0.3.23"]
+                 ;; Force versions to fix prod-mode compilation
+                 [cljsjs/react-with-addons "15.4.0-0"]
```

Comment:
Hmm. I'm not aware of differences in optimization -- seeing it documented neither in the README for [cljsjs/react](https://github.com/cljsjs/packages/tree/master/react) or in [the upstream addons docs](https://facebook.github.io/react/docs/addons.html). If you have a pointer re: that understanding, I wouldn't mind following it up -- that said, AFAIK react-with-addons wasn't used at all until it started being pulled in as a dependency by (older releases of) data-frisk. I made it a dependency at an interim point in this PR, before realizing that we could just upgrade data-frisk and moot the issue.

...actually, per Odinodin/data-frisk-reagent#10, it looks like none of the experimental bits included in the addons package were actually used even in the older release.

BTW, part of my concern here is that React is a big enough piece that if we ended up with different versions (not just with-addons vs base react, but, say, 15.4.0 vs 15.0.0), there could be substantially different behavior between the two modes, to the point that I'd start to worry about whether we could see bugs in production mode that couldn't be reproduced in dev mode.